### ISSUE TITLE
ProfileView + PyPlot crashes Julia

### ISSUE THREAD

**Issue by bjncn** [timholy/ProfileView.jl#39]
~~~
import ProfileView
import PyPlot
PyPlot.plot(rand(10))
~~~
creates a systematic crash
~~~
WARNING: could not import Base.help into PyCall
Fatal Python error: PyEval_RestoreThread: NULL tstate

signal (6): Aborted
gsignal at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
abort at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x7f35906d4ffe)
PyEval_RestoreThread at /usr/lib/x86_64-linux-gnu/libpython2.7.so (unknown line)
unknown function (ip: 0x7f358db15ed3)
TclInvokeStringCommand at /usr/lib/x86_64-linux-gnu/libtcl8.6.so (unknown line)
TclNRRunCallbacks at /usr/lib/x86_64-linux-gnu/libtcl8.6.so (unknown line)
unknown function (ip: 0x7f35927cee00)
TclServiceIdle at /usr/lib/x86_64-linux-gnu/libtcl8.6.so (unknown line)
Tcl_DoOneEvent at /usr/lib/x86_64-linux-gnu/libtcl8.6.so (unknown line)
tcl_doevent at /home/ben/.julia/Tk/src/tkwidget.jl:20
tcl_doevent at /home/ben/.julia/Tk/src/tkwidget.jl:20
jlcall_tcl_doevent_21879 at  (unknown line)
jl_apply_generic at /usr/bin/../lib/x86_64-linux-gnu/julia/libjulia.so (unknown line)
anonymous at task.jl:632
unknown function (ip: 0x7f37af2f8308)
unknown function (ip: (nil))
Aborted (core dumped)
~~~

on my machine ( Version 0.4.0-dev+6971 (2015-08-25 22:51 UTC) Commit c7809f8 (5 days old master) on ubuntu).

Not sure if related to #31.

**Comment by timholy** [comment]
That Tk bug was fixed in https://github.com/JuliaLang/Tk.jl/commit/b40081f47dd916d2fdd48c4e05590b27ce566d36, so this is different.

I'm trying to get to the point of releasing a version based on Gtk. Still a few other issues to work through, unfortunately.

**Comment by timholy** [comment]
Presumably fixed by #38
